docs/contributing.md

```diff
@@ -64,6 +72,38 @@ If there is a new feature you'd like to see in `nearai`, please open an issue an
 3. Provide a *code snippet* that demonstrates the feature usage.
 4. If the feature is related to a paper, please include a link.
 
+## Contribute Documentation
+
+If you discover any errors or omissions in our documentation, please open an issue and describe:
+
+* Which explanation or code snippet is incorrect
+* What concept is not clear or missing
+* If you know, what would be the correct explanation or code snippet
+
+If you think you can contribute a fix for the issue, please feel free to open a Pull Request.
+
+To preview your changes locally, you will need to install all the dependencies for the documentation, particularly:
+
+- [`mkdocs`](https://www.mkdocs.org/)
+- [`mkdocs-material`](https://squidfunk.github.io/mkdocs-material/)
+- [`mkdocs-autorefs`](https://pypi.org/project/mkdocs-autorefs/)
+- [`mkdocs-minify-plugin`](https://github.com/byrnereese/mkdocs-minify-plugin)
+- [`mkdocsstrings`](https://mkdocstrings.github.io/)
+
+!!! failure inline end "Cairo Graphics"
+
+      If you encounter a problem with `cairo`, please follow the [mkdocs-material Requirements Guide](https://squidfunk.github.io/mkdocs-material/plugins/requirements/image-processing/#cairo-graphics)
+
+All these dependencies can be installed with `pip`:
```

Comment:
I believe these are all in a `docs` group and can be installed with `python3 -m pip install -e [docs]` or on mac `python3 -m pip install -e \[docs\]`

Reply:
tried it just now and didn't work, where is the docs group defined? maybe I am in the wrong folder

Reply:
It's defined in `pyproject.toml` but it's in Poetry format, not Pip, so the install command is `poetry install --with docs`.